translations/ja.json

```diff
@@ -8,28 +8,23 @@
 	"'none' driver does not support 'minikube podman-env' command": "'none' ドライバーは 'minikube podman-env' コマンドをサポートしていません",
 	"'none' driver does not support 'minikube ssh' command": "'none' ドライバーは 'minikube ssh' コマンドをサポートしていません",
 	"'none' driver does not support 'minikube ssh-host' command": "'none' ドライバーは 'minikube ssh-host' コマンドをサポートしていません",
-	"'{{.driver}}' driver reported a issue that could affect the performance.": "'{{.driver}}' ドライバーが性能に影響しうる問題を報告しました。",
```

Comment:
This string looks like to be removed, this is because this string is removed from [original strings file](https://github.com/kubernetes/minikube/blob/master/translations/strings.txt), right?

Reply:
@atoato88 right. See strings.txt for details.

Reply:
Could you write some outlines for this PR at description, if you are convenient?
That may be followings:
- remove some target texts from `ja.json` because that target texts have already removed from [original file](https://github.com/kubernetes/minikube/blob/master/translations/strings.txt).
- update xxx from yyy (something like that?

Reply:
Done. PTAL

Reply:
Thank you to update, you add comments as commit message, right?
My comment above is "please write outlines to description on this PR", so could you add same one to top of this PR?
This is because reviewer and other member can know "Why is this part updated on this PR" quickly, I think.